tests/API/OVAL/unittests/test_state_check_existence.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+result=`mktemp`
+
+set -e
+set -o pipefail
+
+cat > /tmp/foo.txt <<EOF
```

Comment:
I always try to use `mktemp` where possible, but this is not the case. The name of the file appears in the attached OVAL content used in a textfilecontent54_test. ( `tests/API/OVAL/unittests/state_check_existence.xml` )